vcsclient/vcsclient.proto

```diff
@@ -38,6 +38,12 @@ message GetFileOptions {
 	// the nearest line boundaries on both sides. It is only valid if StartByte and
 	// EndByte are specified.
 	bool full_lines = 4 [(gogoproto.moretags) = "url:\",omitempty\""];
+
+	// FullTree only applies if the returned entry is a directory. It will
+	// return the full file tree of the host repository, recursing into all
+	// sub-directories. The base path will be the root of the repository,
+	// not the requested path!
+	bool full_tree = 5 [(gogoproto.moretags) = "url:\",omitempty\""];
```

Comment:
I think that this flag is usually called "recursive" in other tools, such as `ls --recursive`, `aws ls --recursive`, `s3cmd ls --recursive`, etc. So, rename from "full tree" to "recursive"?